apps/web/components/App.tsx

```diff
@@ -53,6 +54,7 @@ export default function App({
   email: string; // required
   tos?: string;
   privacy?: string;
+  multipleInstall?: AppType["multipleInstall"];
```

Comment:
This should be just `boolean`, right? 😄

Reply:
Yes, but with this, in the future if the value change you only have to change it on the origin.